students/jemielity_kamil/lesson_07_string_datetime/table_printer.py

```diff
@@ -0,0 +1,22 @@
+def print_table(data):
+    col_width = []
+    for i in range(len(data)):
+        maximum = 0
+        for j in range(len(data[0])):
+            if len(data[i][j]) > maximum:
+                maximum = len(data[i][j])
```

Comment:
Try to avoid code duplication, data[i][j] might be assigned to dedicated variable